### PR TITLE
Ad469x fmcx iio_support

### PR DESCRIPTION
**Pull request by mchindri**

### CODE REVIEW

File: projects/ad469x_fmcz/src/ad469x_iio.h [hunk 1]
```diff
@@ -0,0 +1,45 @@
+#ifndef AD469X_IIO
+#define AD469X_IIO
+
+#include "iio_types.h"
+#include "ad469x.h"
+
+
+struct scan_type ad469x_scan_type = {
```
Comment (cristiantpop) [inline projects/ad469x_fmcz/src/ad469x_iio.h:8]: `ad469x_scan_type` structure is also declared in c file. Probably we should remove from here

File: projects/ad469x_fmcz/src/ad469x_iio.h [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+	.sign = 'u',
+	.realbits = 19,
+	.storagebits = 32,
+	.shift = 0,
+	.is_big_endian = false
+};
+
+static struct iio_channel ad469x_ch0 = {
```
Comment (cristiantpop) [inline projects/ad469x_fmcz/src/ad469x_iio.h:16]: Same with other structures

**Comment by mchindri** [comment]
v2:
 - Use num_slots instead of creating new ch_mask
 - remove ad469x_iio.h because it was not used
 - other small fixes

File: projects/ad469x_fmcz/src/ad469x_fmcz.c [hunk 3]
```diff
@@ -175,15 +176,19 @@ int main()
 		if (ret != SUCCESS)
 			return ret;
 
-		for (i = 0; i < AD469x_EVB_SAMPLE_NO; i += 3)
 			printf("ADC sample: %"PRIu32", ch0: %"PRIu32", ch1:%"PRIu32", temp: %"PRIu32"\n",
+		for (i = 0; i < AD469x_EVB_SAMPLE_NO * 3; i += 3)
```
Comment (cristiantpop) [inline projects/ad469x_fmcz/src/ad469x_fmcz.c:312]: For in printf?!

File: projects/ad469x_fmcz/src/ad469x_fmcz.c [hunk 4]
```diff
@@ -129,6 +210,7 @@ int main()
 		.dev_id = ID_AD4696, /* dev_id */
 		.dcache_invalidate_range =
 		(void (*)(uint32_t, uint32_t))Xil_DCacheInvalidateRange,
+		(void (*)(uint32_t, uint32_t))0,
```
Comment (cristiantpop) [inline projects/ad469x_fmcz/src/ad469x_fmcz.c:213]: Add explicit structure field

**Comment by mchindri** [comment]
v3:
 - Fix printf 
 - Remove extra line added to init_param

File: projects/ad469x_fmcz/src/ad469x_fmcz.c (hunk 4 above)
Comment (mchindri) [reply projects/ad469x_fmcz/src/ad469x_fmcz.c]: removed

File: projects/ad469x_fmcz/src/ad469x_fmcz.c (hunk 3 above)
Comment (mchindri) [reply projects/ad469x_fmcz/src/ad469x_fmcz.c:311]: solved

**Review by cristiantpop — APPROVED** [review]
LGTM

**Review by amiclaus — APPROVED** [review]

**Pull request merged**